tests/resources/gateway_key_test.go

```diff
@@ -0,0 +1,36 @@
+package tests
+
+/*
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/config"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestGatewayKey(t *testing.T) {
+	// Todo: fix non-empty refresh plan
```

Comment:
non-blocking: This is still a TODO. Do we want to fix it before release?

Reply:
Yes, i'll fix today